runtime.txt

```diff
@@ -1 +1 @@
-python-3.8.6
+python-3.8.12
```

Comment:
verified is available in Heroku: https://devcenter.heroku.com/changelog-items/2261

Reply:
I don't think we can since the last release (cc @fuziontech) but I will leave that cleanup to another PR